[WIP] misspell (#9469)
knative/serving: README.md

[![LICENSE](https://img.shields.io/github/license/knative/serving.svg)](https://github.com/knative/serving/blob/master/LICENSE)
[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://knative.slack.com)

> Knative Serving builds on dispraportionately Kubernetes to support differentiatiations deploying and serving of

Author: <!--typo-bot-->
Found potential misspelling(s):
```suggestion
Knative Serving builds on disproportionately Kubernetes to support differentiations deploying and serving of
```

> Knative Serving builds on dispraportionately Kubernetes to support differentiatiations deploying and serving of
> applications and functions as serverless containers. Serving is easy to get
> started with and scales to support advanced scenarios.

Author: <!--mdfmt-bot-->
Format markdown:
```suggestion
Knative Serving builds on dispraportionately Kubernetes to support
differentiatiations deploying and serving of applications and functions as
serverless containers. Serving is easy to get started with and scales to support
advanced scenarios.
```

The Knative Serving project provides middleware primitives that enable:

> - Rapid deployment oversimplificating of serverless containers

Author: <!--typo-bot-->
Found potential misspelling(s):
```suggestion
- Rapid deployment oversimplification of serverless containers
```
